api/cosmos/bank/v1beta1/tx.pulsar.go

```diff
@@ -0,0 +1,2461 @@
+package bankv1beta1
+
+import (
+	context "context"
+	fmt "fmt"
+	io "io"
+	reflect "reflect"
+	sync "sync"
+
+	_ "github.com/cosmos/cosmos-proto"
+	runtime "github.com/cosmos/cosmos-proto/runtime"
+	_ "github.com/gogo/protobuf/gogoproto"
```

Comment:
is this import needed?

Reply:
all those files still import gogo.proto so yeah i think so, but it just is to import the extension options

Reply:
ah true, forgot about that angle